src/main/java/Dude.java

```diff
@@ -0,0 +1,204 @@
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class  Dude {
```

Comment:
`public class  Dude` In your code, there are 2 spaces between class and Dude. Consider to follow the coding standard that this is a space exactly.